src/Components/Header.jsx

```diff
@@ -0,0 +1,29 @@
+import logo from "../assets copy/images/logo.svg"
+import NavMobile from "./NavMobile"
+import "../style/header.css"
+import NavDesktop from "./NavDesktop"
+import { useState, useEffect } from "react";
+
+export default function Header() {
+  const [isMobile, setMobile] = useState(window.innerWidth < 1000);
```

Comment:
Can we use 600 or perhaps even 650px, to make sure the nav looks good? If we use 376 it will be hard to get it to look good I think? Of course we can change font-size but we don't want the text to be too small. 

This is 500px
![Skärmbild 2023-06-19 125311](https://github.com/akickia/news-homepage/assets/95421577/eea2590b-c5d3-4b16-b70f-da2747a4200d)